h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
> Or we could officially take away this library, which would mean displaying our own URL on the source, so that when someone looks for the upstream, it finds us.

I mean, we could officially take **over** this library.